### DISCUSSION TITLE
Path Issue

### DISCUSSION THREAD

**Discussion by sahiljadhav123** [chezou/tabula-py#394] [Q&A] [answered]
Hello Sir,
I am reading Pdf from Tabula library but it takes Static Path from my environment of system for tabula.read_pdf 
can we have other parameter to provide Static path of jvm.dll that provide and it runs from provided path 
Thanks For reply....

**Comment by chezou** [comment, marked as answer]
As described in the document, the Windows registry manages PATH to JAVA. This is outside the scope of tabula-py. 
https://tabula-py.readthedocs.io/en/latest/getting_started.html#get-tabula-py-working-windows-10

You can switch by setting JAVA_HOME like https://stackoverflow.com/a/26993233.

Anyway, it is not tabula-py scope, so I can't answer anything additional.

**Reply by sahiljadhav123** [reply]
Okay, Thanks Sir.....

**Reply by sahiljadhav123** [reply]
one last question is it only work for windows 10 ?????
